controllers/bootstrap/init.go

```diff
@@ -432,3 +480,35 @@ func (b *Bootstrap) deleteSubscription(name, namespace string) error {
 
 	return nil
 }
+
+// Get ConfigMap of Common Services Maps
+func (b *Bootstrap) getCmOfMapCs() (*corev1.ConfigMap, error) {
+	cmName := constant.CsMapConfigMap
+	cmNs := "kube-public"
+	cm := &corev1.ConfigMap{}
+	err := b.Reader.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: cmNs}, cm)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			klog.Errorf("NotFound ConfigMap: %s %s", cmNs, cmName)
+			return nil, nil
+		}
+		klog.Errorf("Failed to get ConfigMap %s %s", cmNs, cmName)
+		return nil, err
+	}
+	return cm, nil
+}
+
+// GetMasterNs gets MasterNamespaces of deploying Common Services
+func (b *Bootstrap) GetMasterNs() (string, bool) {
+	operatorNs, err := util.GetOperatorNamespace()
+	if err != nil {
+		klog.Errorf("Getting operator namespace failed: %v", err)
+		return "", false
+	}
+
+	cm, _ := b.getCmOfMapCs()
+	if masterNs, found := cm.Data[operatorNs]; found {
```

Comment:
```
data:
common-service-maps:
  - requested-from-namespace: cp4iforFord
    map-common-service-namespace: common-services-ford
  - requested-from-namespace: cp4iforToyota
    map-common-service-namespace: common-services-toyota
  map-default: ibm-common-services
 ```

Please follow the above example to add descriptive keys into the configmap to make it more readable.

Reply:
How about
```
data:
  common-service-maps.yaml: |
    namespaceMapping:
    - requested-from-namespace: cp4iforFord
      map-common-service-namespace: common-services-ford
    - requested-from-namespace: cp4iforToyota
      map-common-service-namespace: common-services-toyota
    defaultNs: ibm-common-services
```

Reply:
@horis233 
It looks good to me. I will follow this format to revise the code